kms-message/src/kms_kmip_result_status_private.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef KMS_KMIP_RESULT_STATUS_PRIVATE_H
+#define KMS_KMIP_RESULT_STATUS_PRIVATE_H
+
+#include "kms_message/kms_message_defines.h"
+
+#define KMS_XMACRO                                           \
```

Comment:
Should this be preceeded by a check/assertion that `KMS_XMACRO` has not already been defined?